### PR TITLE
css-grid-classes

### PR DESCRIPTION
**Pull request by KennethDelaCruz**
image

### CODE REVIEW

File: css-grid-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.container {
+  margin: auto;
+  width: 1000px;
+}
+
+.row {
+  display: flex;
+  align-items: center;
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css]: If you remove this...

File: css-grid-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.column-full {
+  display: flex;
+  align-self: flex-start;
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css]: You won't need this.

File: css-grid-classes/styles.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+.column-one-third {
+  margin: 5px;
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css]: If you're going to use an exact width like this, this margin will break your content if you ever set the row to `flex-wrap: wrap`. This should be removed.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Kenn, great work so far! Please see my notes

File: css-grid-classes/styles.css (hunk 3 above)
Comment (KennethDelaCruz) [reply css-grid-classes/styles.css]: Got it, Thank you Cody!

I will adjust my code for all the changes

**Review by codingcodymiller — APPROVED** [review]
Nicely done!! 👍

**Pull request merged**